integreat_cms/cms/views/region_condition/region_condition_actions.py

```diff
@@ -48,6 +48,11 @@ class RegionConditionResource(resources.ModelResource):
 
     num_outdated_pages = fields.Field(column_name=_("Number of outdated pages"))
 
+    has_translation_package_been_booked = fields.Field(
+        column_name=_("Booked MT budget"),
+        attribute="mt_budget_booked",
```

Comment:
```suggestion
```

Reply:
I guess `attribute` corresponds to a field or a property of the model. If we change it to `mt_budget`, not the booked budget (50k, 100k, etc) is shown in the column, but the property `mt_budget` which is the number of words calculated with `mt_renewal_month` and `mt_midyear_start_month`. 

In my opinion this column should show which MT budget was booked, not how much budget each region has this year, as it was shown there before the change whether the add-on was booked or not.